### DISCUSSION TITLE
PfSense via Elastic Agent.

### DISCUSSION THREAD

**Discussion by givilleneuve** [pfelk/pfelk#353] [General]
Hi guys,

I have noticed that the PfSense is now available through the Fleet using Elastic Agent.
Have any of you used that way?

I have attached to the policy, but it doesn't seen to listen nothing in the default port.

Thank you,

**Comment by a3ilson** [comment]
Can you clarify where this agent is? I just checked and didn't note any pfSense agent
![Screen Shot 2021-11-05 at 6 12 06 PM](https://user-images.githubusercontent.com/16884679/140584233-2629d192-2d21-4820-b2b2-fbd1a6dc82fa.png)
:

**Reply by a3ilson** [reply]
Sorry...I was running an older version...it's there...they utilized this repo to build the agent but no mention of this project... and it doesn't work as well - needs some work but once it's done, I'll defer to Elastic continue for continued support.
![Screen Shot 2022-02-21 at 8 18 34 PM](https://user-images.githubusercontent.com/16884679/155045962-924ed7d6-f325-4ea5-9c75-202dea9c1596.png)
